Tests/Packet++Test/Tests/LdapTests.cpp

```diff
@@ -626,19 +748,21 @@ PTF_TEST_CASE(LdapCreationTest)
 		pcpp::Packet searchResultEntryPacket(&rawPacket1);
 
 		std::vector<pcpp::LdapAttribute> attributes = {
-			{"objectclass", {"inetOrgPerson", "organizationalPerson", "person", "top"}},
-			{"sn",          {"Young"}},
-			{"cn",          {"b.smith"}},
-			{"givenname",   {"Beatrix"}}
+			{ "objectclass", { "inetOrgPerson", "organizationalPerson", "person", "top" } },
+			{ "sn",          { "Young" }			                                      },
+			{ "cn",          { "b.smith" }			                                    },
+			{ "givenname",   { "Beatrix" }			                                    }
```

Comment:
ditto: a mix of tabs and spaces, which is a known bug of clang-format. We should probably disable formatting here

Reply:
Disable the format.